src/Command/CreateFlagHandler.php

```diff
@@ -26,18 +28,26 @@ class CreateFlagHandler
      */
     protected $posts;
 
+        /**
```

Comment:
There are a few formatting errors in this file's diff. :wink:

Reply:
That'll teach me for rushing! Corrected...